test/server/core/abilities/keyword/Sentinel.spec.js

```diff
@@ -0,0 +1,51 @@
+describe('Sentinel keyword', function() {
+    integration(function() {
+        describe('When a unit with the Sentinel keyword', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['liberated-slaves'],
+                    },
+                    player2: {
+                        groundArena: ['echo-base-defender'],
+                        spaceArena: ['system-patrol-craft']
+                    }
+                });
+            });
+
+            it('is in play, it must be targeted by an attack', function () {
+                this.player1.clickCard(this.liberatedSlaves);
+                expect(this.liberatedSlaves.exhausted).toBe(true);
+                expect(this.p2Base.damage).toBe(0);
+                expect(this.liberatedSlaves.damage).toBe(4);
+                expect(this.echoBaseDefender).toBeInLocation('discard');
+            });
+        });
+
+        describe('When two units with the Sentinel keyword', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['liberated-slaves'],
+                    },
+                    player2: {
```

Comment:
In addition to some non-sentinel units can we add a space sentinel just to be sure it's not getting including by accident